<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterTicketsByOne</class>
 <widget class="QWidget" name="FilterTicketsByOne">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="WdtContainer" name="wc" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Fuel</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="lbBack">
        <property name="text">
         <string>Back date</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="C5DateEdit" name="deSold1">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="FilterName" stdset="0">
         <string>datesold1</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="C5DateEdit" name="deIssue2">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="FilterName" stdset="0">
         <string>dateissue2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Ticket</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lbIssue">
        <property name="text">
         <string>Issue date</string>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="C5DateEdit" name="deBack2">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="FilterName" stdset="0">
         <string>dateback2</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="C5LineEdit" name="lineEdit_5">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="lvalue" stdset="0">
         <string>state</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>352</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="3">
       <widget class="C5LineEdit" name="lineEdit_2">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="lvalue" stdset="0">
         <string>ticket_type</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lbSold">
        <property name="text">
         <string>Sold date</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="C5LineEdit" name="leFuel">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="FilterName" stdset="0">
         <string>fuel</string>
        </property>
        <property name="lkey" stdset="0">
         <string>fuel</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="C5DateEdit" name="deIssue1">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="FilterName" stdset="0">
         <string>dateissue1</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="C5DateEdit" name="deBack1">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="FilterName" stdset="0">
         <string>dateback1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="C5LineEdit" name="lineEdit_3">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="lvalue" stdset="0">
         <string>fuel</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>State</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="C5LineEdit" name="leTicket">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="FilterName" stdset="0">
         <string>ticket_type</string>
        </property>
        <property name="lkey" stdset="0">
         <string>ticket_type</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="C5LineEdit" name="leState">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="FilterName" stdset="0">
         <string>state</string>
        </property>
        <property name="lkey" stdset="0">
         <string>state</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Partner</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="C5LineEdit" name="lineEdit_4">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="lvalue" stdset="0">
         <string>partners</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="C5LineEdit" name="lePartner">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="FilterName" stdset="0">
         <string>partner</string>
        </property>
        <property name="lkey" stdset="0">
         <string>partners</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="C5DateEdit" name="deSold2">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="FilterName" stdset="0">
         <string>datesold2</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Payment type</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="C5LineEdit" name="lePayment">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="FilterName" stdset="0">
         <string>payment</string>
        </property>
        <property name="lkey" stdset="0">
         <string>payment</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="C5LineEdit" name="lineEdit_6">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="lvalue" stdset="0">
         <string>payment</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>WdtContainer</class>
   <extends>QWidget</extends>
   <header location="global">wdtcontainer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>C5DateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5dateedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leFuel</tabstop>
  <tabstop>leTicket</tabstop>
  <tabstop>lePartner</tabstop>
  <tabstop>deIssue1</tabstop>
  <tabstop>deIssue2</tabstop>
  <tabstop>deSold1</tabstop>
  <tabstop>deSold2</tabstop>
  <tabstop>deBack1</tabstop>
  <tabstop>deBack2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_4</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
